app/models/company.rb

```diff
@@ -9,6 +9,7 @@ class Company < ActiveRecord::Base
   has_many :options
   has_many :responses
   has_many :user_settings
+  has_many :activities
```

Comment:
please add dependent: :destroy on all associations in company

Reply:
done